back-end/src/models/contactsLog.model.js

```diff
@@ -0,0 +1,20 @@
+const mongoose = require('mongoose');
```

Comment:
Use .ts extension (TypeScript). If you don't know TypeScript, can use JavaScript within the .ts file.